Fluent/Post_Process.ipynb

```diff
@@ -6,127 +6,147 @@
     "\n",
-    "Create a new cell using `Edit/Split Cell` path and insert the following lines in the newly created cell and compile it, while in `Code` environment rather than `Markdown` environment.\n",
+    "Fig. 2 visualizes the contours of vorticity magnitude within the CFD domain. Flow enters the domain from left to right. Due to the imposed no-slip boundary conditions over the surface of the cylinder, the velocity instantly decreases to zero on the nose of the cylinder and a laminar boundary layer starts to form and grow around the cylinder’s surface. Moving slightly further downstream at the top and bottom edges of the cylinder, where the formed laminar boundary layer is exposed to the free stream velocity, a shear layer will form and result in negatively and positively accelerated flow regions at these two points. These two regions can be thought of as continuous vortices along the cylinder’s surface. In the case study with a stationary cylinder (i.e. $\\alpha = 0$) the body of the cylinder, as a blunt body, is blocking the incoming flow, a decelerated region of flow would form right at the back of the cylinder, as shown in blue. This area is the place in which the flow recirculates and two counter rotating vortices, as a result of flow moving over the top and bottom curved surfaces of the cylinder, would form here. At low Reynolds numbers (i.e. below Re=100) this region would stay attached to the cylinder. However, in this study case the Reynolds number is great enough to cause unsteady vortex shedding from the cylinder. This phenomena is known as the von Karman effect and it is a well-known physical process. For the case study with a fixed cylinder (i.e. $\\alpha = 0$) it is observed that the vortex shedding has a symmetric pattern. \n",
     "\n",
-    "```\n",
-    "from IPython.display import YouTubeVideo\n",
-    "YouTubeVideo(\"kyAJiOXQIq4\")\n",
-    "```"
+    "Increasing the rotational speed (i.e. $\\alpha$) of the cylinder would increase the strength of vorticity generated on the cylinder surface increases. This is accompanied by an increase in the thickness of the region with closed streamlines around the cylinder. For vortex shedding to occur, vorticity of large enough strength must be carried to the outer flow, outside closed streamlines, via diffusion. This region of vorticity, in the outer flow, must also be a sufficiently large area of slowly moving, preferably recirculating fluid. Under these conditions the vorticity can pile up in a certain region of the flow and vortex shedding can take place. For $\\alpha = 0$, a large region behind the cylinder is associated with slowly moving fluid. The size of this region reduces, rapidly, with increase in rotational speed of the cylinder. This explains the disappearance of vortex shedding for $\\alpha = 2$ as visualized in the animation shown in Fig. 3."
```

Comment:
Your explanation in this paragraph is very wage. Avoid using everyday terms such as "sufficiently large" or "slowly moving". There are vorticity magnitude and Reynolds number definition to technically elaborate the physical process of interest. Use those definitions and terminologies to explicitly discuss and explain physical phenomena.

Re-write this paragraph.